pkg/ext-proc/handlers/request.go

```diff
@@ -66,17 +72,17 @@ func (s *Server) HandleRequestBody(reqCtx *RequestContext, req *extProcPb.Proces
 		rb["model"] = llmReq.ResolvedTargetModel
 		requestBody, err = json.Marshal(rb)
 		if err != nil {
-			klog.V(logutil.DEFAULT).ErrorS(err, "Error marshaling request body")
+			logger.V(logutil.DEFAULT).Error(err, "Error marshaling request body")
 			return nil, fmt.Errorf("error marshaling request body: %v", err)
 		}
-		klogV.InfoS("Updated request body marshalled", "body", string(requestBody))
+		loggerVerbose.Info("Updated request body marshalled", "body", string(requestBody))
 	}
 
-	targetPod, err := s.scheduler.Schedule(llmReq)
+	targetPod, err := s.scheduler.Schedule(ctx, llmReq)
 	if err != nil {
 		return nil, fmt.Errorf("failed to find target pod: %w", err)
 	}
-	klogV.InfoS("Target model and pod selected", "model", llmReq.ResolvedTargetModel, "pod", targetPod)
+	loggerVerbose.Info("Target model and pod selected", "model", llmReq.ResolvedTargetModel, "pod", targetPod)
```

Comment:
This should be `DEFAULT` log level. Can we also log the model (not just target), something like:

```
logger.V(logutil.DEFAULT).("Request handled", "model", llmReq.Model, "targetModel", llmReq.ResolvedTargetModel, "endpoint", targetPod)
```